lib/aws/eks.ts

```diff
@@ -408,139 +408,167 @@ export class EksStack {
         // }
       },
     });
-
+    
+    const sdk_version = "0.27.2";
     const hypersChart = cluster.addHelmChart("HyperswitchServices", {
-      chart: "hyperswitch-helm",
-      repository: "https://juspay.github.io/hyperswitch-helm",
+      chart: "hyperswitch-stack",
+      repository: "https://juspay.github.io/hyperswitch-helm/v0.1.2",
       namespace: "hyperswitch",
       release: "hypers-v1",
```

Comment:
Can we change the release name to something like `hyperswitch-0.1.2`? Would `hypers-v1 `suffice?